mbuild/formats/hoomdxml.py

```diff
@@ -78,6 +78,13 @@ def write_hoomdxml(structure, filename, ref_distance=1.0, ref_mass=1.0,
         forcefield = False
     xyz = np.array([[atom.xx, atom.xy, atom.xz] for atom in structure.atoms])
 
+    # Check if we need to shift things to -L/2, L/2
+    box_max = structure.box[:3]/2.
```

Comment:
The repeated code from the GSD writer got me thinking that maybe we should have a `Compound.wrap` method that takes a box as an argument. I realize here in the writer we are operating on a ParmEd `Structure` (which might already has a similar method?), but maybe the wrapping is something we can handle within `Compound.save` prior to converting to a `Structure`.

Reply:
I'm in favor of moving the shifting code into a function, somewhere in `utils` perhaps? the `.wrap` scares me as there are a million pitfalls for wrapping things into a pbc box.

Reply:
What about `shift_coords` in `utils/geometry.py`? Or  `utils/conversion.py`?

Reply:
Yeah, putting together a `wrap` method might be better served as a separate PR down the line, as it will take some work handling all of the edge cases. For now, moving the code to `utils/geometry.py` sounds good.